packages/cli/src/__tests__/plugin.test.js

```diff
@@ -7,17 +17,24 @@ import { getConfig, getPaths } from '@redwoodjs/project-config'
 import * as pluginLib from '../lib/plugin'
 import { loadPlugins } from '../plugin'
 
-jest.mock('fs')
-jest.mock('@redwoodjs/project-config', () => {
+vi.mock('fs-extra', async () => {
+  const memfs = await import('memfs')
```

Comment:
This is great! If you want to continue replacing our custom fs mocks with memfs that'd be great! 
(But also don't feel like you have to if you don't want to)

Reply:
Yeah, the motivation was simply that we had tests for our custom fs mock within a `fs.test.js` file. Those were failing and it would be work to get them passing again. Retaining our custom mock and using a more robust community offering felt more sensible so happy to.